scylla/src/transport/load_balancing/round_robin.rs

```diff
@@ -34,10 +34,17 @@ impl LoadBalancingPolicy for RoundRobinPolicy {
         cluster: &'a ClusterData,
     ) -> Box<dyn Iterator<Item = Arc<Node>> + Send + Sync + 'a> {
         let index = self.index.fetch_add(1, ORDER_TYPE);
-
         let nodes_count = cluster.all_nodes.len();
         let rotation = super::compute_rotation(index, nodes_count);
         let rotated_nodes = super::slice_rotated_left(&cluster.all_nodes, rotation).cloned();
+        trace!(
+            nodes = rotated_nodes
+                .clone()
+                .map(|node| node.address.to_string())
+                .collect::<Vec<String>>()
+                .join(",")
+                .as_str()
+        );
```

Comment:
Might be good to include some information that this is related to a round robin policy (in DC aware policy you already print "DC Aware").